test/proxy/utils/Initializable.test.js

```diff
@@ -213,7 +213,7 @@ contract('Initializable', function () {
     it('old and new patterns in good sequence', async function () {
       const ok = await DisableOk.new();
       await expectEvent.inConstruction(ok, 'Initialized', { version: '1' });
-      await expectEvent.inConstruction(ok, 'Initialized', { version: '255' });
+      await expectEvent.inConstruction(ok, 'Initialized', { version: (2n ** 64n - 1n).toString() }); // MAX_UINT64
```

Comment:
we should consider generalizing this syntax.

Reply:
I used `web3.utils.toBN(1).shln(64).subn(1).toString()` instead because I think we've previously agreed on a similar one in the ERC2771Forwarder (but for `uint48`), does this work?